conda_forge_tick/utils.py

```diff
@@ -1306,7 +1306,7 @@ def change_log_level(logger, new_level):
         logger.setLevel(saved_logger_level)
 
 
-def run_command_hiding_token(args: list[str], token: str) -> int:
+def run_command_hiding_token(args: list[str], token: str, **kwargs) -> int:
```

Comment:
If you want to do this, you need to do some validation to avoid "duplicate" kwarg errors:

```
>>> from subprocess import run
>>> def runn(*args, **kwargs):
...   return run(*args, text=True, **kwargs)
...
>>> runn(["echo", "True"])
True
CompletedProcess(args=['echo', 'True'], returncode=0)
>>> runn(["echo", "True"], text=False)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 2, in runn
TypeError: subprocess.run() got multiple values for keyword argument 'text'
```

Make sure kwargs does not include `text`, `stdout` or `stderr`. Or override the defaults with the ones in kwargs, but do handle them somehow.